Mathlib/CategoryTheory/Whiskering.lean

```diff
@@ -113,6 +113,26 @@ def Functor.FullyFaithful.whiskeringRight {F : D ⥤ E} (hF : F.FullyFaithful)
         simp only [map_comp, map_preimage]
         apply f.naturality }
 
+@[simp]
+theorem whiskeringLeft_obj_id : (whiskeringLeft C C E).obj (.id _) = .id _ :=
+  rfl
+
+@[simp]
```

Comment:
I am not completely sure we want to set the `simp` attribute here.

Reply:
Are you referring to the `whiskeringLeft_obj_conp` only? I'm not sure about that one was well (or about its direction), I'll look at a few similar examples... Have you seen my questions in the PR comment above?

Reply:
I am referring to both `_comp`. I am ok with the `_id`. You may very well add `...Iso` versions of these as well.
(I do not have a strong opinion of functors from `Cat` to `Cat`. It may go in a separate PR *if there is a need for these*.)

Reply:
I would trust the opinion of @yuma-mizuno about not using the `simp` attribute for the `_id` as well.

Reply:
Thanks!